io/disk/arcconf/arcconf_drive_oper.py

```diff
@@ -0,0 +1,194 @@
+        self.os_drive = self.cmdop_list(cmd)
+
+        if self.diskoverwrite == 'Y' and self.os_drive != "":
+            # Delete a logical drive only if it is not OS drive
+            cmd = "df -h /boot | grep %s" % self.os_drive
+            if process.system(cmd, timeout=300, ignore_status=True,
+                              shell=True) == 0:
+                self.skip("Test Skipped!! OS disk requested for removal")
+
+            self.log.info("Deleting the default logical drive %s" %
+                          (self.logicaldrive))
+            cmd = "echo y | arcconf delete %s logicaldrive %s" % \
+                (self.crtl_no, self.logicaldrive)
+            self.check_pass(cmd, "Failed to delete Logical drive")
+
+    def test(self):
+        """
+        Main function
+        """
+        cmd = "arcconf getconfig %s PD" % self.crtl_no
+        self.check_pass(cmd, "Failed to display PMC disk details")
+        loop_count = 0
+        disk_val = ""
+        # pick only first two disks
+        for disk in self.disk_no[:2]:
+            loop_count += 1
+            disk_val += " %s %s" % (self.channel_no, disk.strip())
+            # test logicaldrive specific options
+            if "setname" in self.option or "setboot" in self.option:
+                if loop_count == 2:
+                    # create logical drive
+                    cmd = "arcconf create %s logicaldrive MAX 0 %s"\
+                        % (self.crtl_no, disk_val)
+                    self.check_pass(cmd, "Failed to run %s" % cmd)
+                    if "setname" in self.option:
+                        cmd = "%s %s %s %s %s" % (self.option,
+                                                  self.crtl_no,
+                                                  self.option_args,
+                                                  self.logicaldrive,
+                                                  self.option_args2)
+                    else:
+                        cmd = "%s %s %s %s" % (self.option,
+                                               self.crtl_no,
+                                               self.option_args,
+                                               self.logicaldrive)
+                    self.check_pass(cmd, "Failed to run %s" % cmd)
+            else:
+                # Run disk specific test for one drive
+                if loop_count == 1:
+                    if "setphy" in self.option:
+                        cmd = "%s %s %s" % (self.option, self.crtl_no,
+                                            self.option_args)
+                    else:
+                        cmd = "%s %s %s %s %s" % (self.option,
+                                                  self.crtl_no,
+                                                  self.option_args,
+                                                  disk_val,
+                                                  self.option_args2)
+                    self.check_pass(cmd, "Failed to run %s" % cmd)
```

Comment:
`self.check_pass(cmd, "Failed to run %s" % cmd)` in lines 134 and 147 can be made into one.